editor/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# app/Dockerfile
+
+FROM python:3.10-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    curl \
+    software-properties-common \
+    git \
+    python3-pip \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY ./ /app/
```

Comment:
Note: It may be possible to ignore the `node_modules` in `components/` to have a lighter image.